bedrock/products/templates/products/vpn/resource-center/landing.html

```diff
@@ -0,0 +1,86 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from 'macros-protocol.html' import call_out, split %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('vpn-resource-center')}}
+{% endblock page_css %}
+
+{% block page_title%}{{ ftl('vpn-resource-center-title') }}{% endblock page_title %}
+
+{% block content %}
+  {{ call_out (
+    title=ftl('vpn-resource-center-whats-the-deal'),
+    desc=ftl('vpn-resource-center-discover-how-they'),
+    class='resource-center-page-header resource-center-hero'
+  )}}
+    {# Category component is currently of the scope of this issue, but still stubbed out for use later #}
+
+    {# {% if category_list|length > 1 %}
+    {# <div class="mzp-c-category-list"> #}
+      {# <h2 class="mzp-c-category-list-title"> #}
+        {# <a href="{{ url('products.vpn.resource-center.landing') }}">View All &#62</a>
```

Comment:
This isn't used yet, but should this be a localized string? (also suggest using a CSS pseudo element for the `&#62` symbol), so it's not part of the string itself.